src/clj/kixi/hecuba/data/sensors.clj

```diff
@@ -158,21 +166,34 @@
 
 (defn encode
   ([sensor]
-     (encode sensor false))
+   (encode sensor false))
   ([sensor remove-pk?]
-     (-> sensor
-         (cond-> (:user_metadata sensor) (user-metadata (:synthetic sensor)))
-         (cond-> remove-pk? (dissoc :device_id :sensor_id)))))
+   (-> sensor
+       (cond-> (:user_metadata sensor) (user-metadata (:synthetic sensor)))
```

Comment:
I'm not sure why there was two cond-> to start off with, I'd rather leave them in their original state know that the one we've added works and doesn't affect any other conditions.